_posts/en/newsletters/2024-12-20-newsletter.md

```diff
@@ -0,0 +1,1231 @@
+offer-compatible payments can use [blinded paths][topic rv routing]
+to prevent the spender from learning the identity of the receiver's
+node.
+
+A [new mining interface][news325 mining] for Bitcoin Core saw
+development with the goal to support miners using the [Stratum v2][topic
+pooled mining] protocol that can be set to allow each miner to select
+their own transactions.  However, Anthony Towns noted earlier in the
+year that independent transaction selection could [raise share validation
+costs][news315 shares] for mining pools.  If those pools responded by
+limiting validation, it could allow a invalid shares attack that's
+similar to the well-known [block withholding attack][topic block
+withholding].  A 2011 proposed solution to withholding attacks was
+discussed, although it would require a challenging consensus change.
+
+<!-- FIXME
+- meh, on-the-fly funding:
+  https://bitcoinops.org/en/newsletters/2024/10/04/#eclair-2861
+- meh, on-the-fly funding:
+  https://bitcoinops.org/en/newsletters/2024/10/04/#eclair-2875
+
+-->
+
+<div markdown="1" class="callout" id="releases">
+
+### Summary 2024: Major releases of popular infrastructure projects
+
+FIXME:harding
+
+</div>
+
+## November
+
+ZmnSCPxj proposed the [SuperScalar design][news327 superscalar] for a
+[channel factory][topic channel factories] design using [timeout
+trees][topic timeout trees] to enable LN users to open channels and
+access liquidity more affordably while maintaining trustlessness. The
+design uses a layered timeout tree that requires the service provider
+pay any costs of putting the tree onchain or lose any funds remaining in
+the tree.  This encourages the service provider to incentivize users to
+cooperatively close their channels to avoid the need to go onchain.  The
+design uses both [duplex micropayment channels][topic duplex
+micropayment channels] and [LN-Penalty][topic ln-penalty] payment
+channels, both of which are possible without any consensus changes.
+Despite its complexity---combining multiple channel types and managing
+offchain state---the design can be implemented by a single vendor
+without requiring major protocol changes.  To support the design,
+ZmnSCPxj later proposed a [pluggable channel factory tweak][news330
+plug] to the LN specification.
+
+<!-- FIXME
+- https://bitcoinops.org/en/newsletters/2024/11/15/#mad-based-offchain-payment-resolution-opr-protocol
+
+- https://bitcoinops.org/en/newsletters/2024/11/29/#lisp-dialect-for-bitcoin-scripting
+- splice:
+  https://bitcoinops.org/en/newsletters/2024/11/29/#core-lightning-7719
+  -->
+
+<div markdown="1" class="callout" id="optech">
+## Summary 2024: Bitcoin Optech
+
+In Optech's seventh year, we published 51 weekly [newsletters][] and
+added 35 new pages to our [topics index][].  Altogether, Optech
+published over 120,000 English words about Bitcoin software research and
+development this year, the rough equivalent of a 350-page book.
```

Comment:
```suggestion
development this year, the rough equivalent of a 500-page book.
```
According to google, the average nonfiction book has about 233 words per page.

Reply:
That's pretty sparse.  I think I'm going to keep the assumption of about 350 words per page, since I believe that's what we've used for comparisons in the past and I think it roughly corresponds to technical books like from O'Reilly (e.g., Mastering Bitcoin 3rd ed is 130,000 words / 400 pages = 325 words per page).